tools/hangar/__snapshots__/test_corpus/valid/bring_local.w_compile_tf-aws.md

```diff
@@ -74,6 +74,9 @@ module.exports = function({  }) {
   class Q {
     constructor({  }) {
     }
+    static async greet(name) {
+      return (require("<ABSOLUTE_PATH>/util.js")["greet"])(name)
```

Comment:
I think we need to avoid absolute paths in the jsified results

Reply:
any specific concerns? Happy to add any more tests to battle test this better

Reply:
I think it could be fair if we say Wing's build outputs are only guaranteed to be stable as long as you always compile in the same directory

(thinking about this issue: https://github.com/winglang/wing/issues/3647)

Reply:
I think it's a desirable quality in general for build results to be as deterministic and hermetic as possible.

Another reason that is more concrete: Absolute paths basically prevent `extern`s from being in a published library.

Reply:
chatted with Mark offline - I think we resolved this is OK for now since we're distributing Wing source files in libraries, so the emitted JS files in `.wing` are skipped when libraries are packaged. We also enforce Wing source files only have relative paths: https://github.com/winglang/wing/blob/8c8124f31c1ec6ad48c98a229a5f628a53037173/libs/wingc/src/parser.rs#L811-L816